src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
```

Comment:
It's a good practice to use semantic tags in your HTML. Consider wrapping your `.stars` divs with a `<section>` tag. This will improve the semantic meaning of your HTML and help screen readers understand your content structure better.